src/components/StateFooter/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Button, Grid, makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+    maxWidth: 1000
```

Comment:
This seems arbitrary. Material UI uses a grid system that increments by 8 to maintain consistency. You should always use numbers divisible by 8.

Reply:
I suppose maxWidth can be handled by App and I remove from here..?

Reply:
Yeah, this might end up causing unwanted inconsistencies with alignment and other positions.